### PR TITLE
Fix: Fixed issue where "Open Files on Windows startup" didn't work

### LINKED ISSUE #13720: Bug: Stuck on loading when starting on Windows logon
### Description

When Files is configured to launch on Windows startup, it fails. It stays stuck on the loading page.

### Steps To Reproduce

Activate launch on Windows startup.

### Requirements

Fix the bug.

### Files Version

3.0.1.0

### Windows Version

Windows 11 Pro 23H2 22635.2700

### Log File

[debug.log](https://github.com/files-community/Files/files/13287860/debug.log)

### PR DESCRIPTION
**Pull request by hishitetsu**
**Resolved / Related Issues**
- [X] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #13720 

**Validation**
How did you test these changes?
- [X] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### CODE REVIEW

File: src/Files.App/MainWindow.xaml.cs [hunk 1]
```diff
@@ -181,6 +181,10 @@ public async Task InitializeApplicationAsync(object activatedEventArgs)
 						await mainPageViewModel.AddNewTabByPathAsync(typeof(PaneHolderPage), fileArgs.Files[index].Path);
 					}
 					break;
+
+				case IStartupTaskActivatedEventArgs startupArgs:
```
Comment (hishitetsu) [inline src/Files.App/MainWindow.xaml.cs:185]: Would it be better to use `default` instead of specifying the interface?

Comment (yaira2) [reply src/Files.App/MainWindow.xaml.cs:185]: We could use both, this way it's clear that they have the same behavior.

**Review by yaira2 — APPROVED** [review]
LGTM

File: src/Files.App/MainWindow.xaml.cs (hunk 1 above)
Comment (yaira2) [reply src/Files.App/MainWindow.xaml.cs:185]: This might also be useful if/when we add an option to open the app in the background.

**Comment by Yabino** [comment]
Sorry guys, it's still happening, the bug is still there.

[https://drive.google.com/file/d/1NAHuQ2D3KT1qu4l7U3CoPsO6y6TS_kyn/view?usp=sharing](url)

**Comment by hishitetsu** [comment]
@Yabino It has only been merged into the dev branch right now. The bug should be fixed after the next update.

**Pull request merged**